dask/array/rechunk.py

```diff
@@ -240,21 +191,38 @@ def rechunk(x, chunks, threshold=DEFAULT_THRESHOLD,
         The graph growth factor under which we don't bother introducing an
         intermediate step.
     block_size_limit: int
-        The maximum block size (in bytes) we want to produce during an
-        intermediate step.
-    """
-    threshold = threshold or DEFAULT_THRESHOLD
-    block_size_limit = block_size_limit or DEFAULT_BLOCK_SIZE_LIMIT
+        The maximum block size (in bytes) we want to produce
```

Comment:
Mention that by default this uses the config value?

Reply:
I've added the name of the default config value in the docstring.  I've also verified that this is mentioned in the docs.